usb/usb_host_msc/test/test_msc.c

```diff
@@ -281,12 +269,14 @@ TEST_CASE("sectors_can_be_written_and_read", "[usb_msc]")
     msc_teardown();
 }
 
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
 TEST_CASE("check_README_content", "[usb_msc]")
```

Comment:
Is anything missing in the current implementation? Why can't we implement and run this test?

Reply:
This TC is only applicable for software based fake device where `README_CONTENTS` are already stored.
The purpose of this TC is to read the contents of file `/usb/README.TXT` from the fake device and match it with expected contents.
This type of fake device is made only for IDF v4.4.

For IDF v5.0 onwards, such fake device is not required as spiflash or sdcard medium is used as storage device. In this case the pupose is solved by TC `write_and_read_file`